include/fc/reflect/typename.hpp

```diff
@@ -41,7 +41,7 @@ namespace fc {
          return n.c_str();  
      } 
   };
-  template<typename T> struct get_typename<flat_set<T>>   
+  template<typename T, typename... A> struct get_typename<flat_set<T, A...>>
```

Comment:
This means two `flat_set`s with same `T` but different compare function will be seen as the same? Looks strange, but perhaps doesn't matter for `get_typename`.

Reply:
`T::type_lt` not `T::type_eq`. Sounds like a reasonable solution to me.

Reply:
Ack! `type_lt` indeed. Oops.